source/upgrade/v3.txt

```diff
@@ -135,4 +135,7 @@ Version 3.0 Breaking Changes
      };
 
      // Serialize the document to JSON using the configured settings
-     var json = document.ToJson(jsonWriterSettings);
\ No newline at end of file
+     var json = document.ToJson(jsonWriterSettings);
+
+- The ``MongoClient`` constructor accepts only one ``Credential`` object instead of an
```

Comment:
Note for reviewer: Passing multiple credentials was deprecated a long time ago. Until this release, trying to pass more than one would throw an exception. This change just makes it clear that the driver will only accept one.